kore/test/Test/Kore/Exec.hs

```diff
@@ -1141,9 +1138,9 @@ matchDisjunctionTest ::
     Pattern RewritingVariableName ->
     [Pattern RewritingVariableName] ->
     SMT (TermLike VariableName)
-matchDisjunctionTest = matchDisjunction DisabledSimplifierX
+matchDisjunctionTest = matchDisjunction
 
 checkFunctionsTest ::
     VerifiedModule Attribute.StepperAttributes ->
     SMT ()
-checkFunctionsTest = checkFunctions DisabledSimplifierX
+checkFunctionsTest = checkFunctions
```

Comment:
We could probably just use the original functions now (`matchDisjunctions`, `checkFunctions`)

Reply:
good point